### DISCUSSION TITLE
Paperless Script does not work

### DISCUSSION THREAD

**Discussion by Belei99** [tteck/Proxmox#2513] [General]
Hi, 
iam on proxmox 8.1.3. i start the shell in the web interface, logedin with root. 
i am copy the paperless install script but nothing happen, no error, nothing...

whats my problem?

**Comment by tteck** [comment]
It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by Belei99** [reply]
Fantastic you are right. I change the fritz.box ip to my pi.hole ip in the dns settings from proxmox and now it installs.
